src/any_agent/tools/mcp/frameworks/google.py

```diff
@@ -51,11 +52,20 @@ class GoogleMCPStdioConnection(GoogleMCPConnection):
 
     async def list_tools(self) -> list["GoogleMCPTool"]:
         """List tools from the MCP server."""
-        self._params = GoogleStdioServerParameters(
+        server_params = GoogleStdioServerParameters(
             command=self.mcp_tool.command,
             args=list(self.mcp_tool.args),
             env=self.mcp_tool.env,
         )
+
+        timeout = self.mcp_tool.client_session_timeout_seconds
+        if timeout is None:
+            self._params = server_params
+        else:
+            self._params = StdioConnectionParams(
+                server_params=server_params,
```

Comment:
I had to go and check the google adk. I am so confused about this API. Like why `SseConnectionParams` has all the args but `StdioConnectionParams` has a nested object with the params 🤷